HW2/HW2EX1/ses-test/cmd.ap.sub_03

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env tcsh
+
+# created by uber_subject.py: version 0.40 (March 30, 2017)
+# creation date: Thu Aug  3 22:18:44 2017
+
+# set data directories
+set top_dir   =                                                             \
+    /Users/timmichaels/Documents/UConn/BIRC/WedJuly26/Non-Linear/playground/HW2
+set anat_dir  = $top_dir/ses-test/anat
+set epi_dir   = $top_dir/ses-test/func
+
+# set subject and group identifiers
+set subj      = sub_03
+set group_id  = test
+
+# run afni_proc.py to create a single subject processing script
+afni_proc.py -subj_id $subj                                                 \
+        -script proc.$subj -scr_overwrite                                   \
+        -blocks tshift align tlrc volreg blur mask scale regress            \
+        -copy_anat $anat_dir/sub-03_ses-test_T1w.nii.gz                     \
+        -tcat_remove_first_trs 0                                            \
```

Comment:
You can change this to `-tcat_remove_first_trs 1` so the first volume gets dropped. Then you don't have to edit the proc script.